package.json

```diff
@@ -32,6 +32,7 @@
     "semver": "^5.3.0",
     "sosa_mongo": "^1.0.3",
     "timebucket": "^0.4.0",
+    "talib": "^1.0.3",
```

Comment:
FYI, Talib doesnt work on Windows, so the PR will break the master for windows.
npm ERR! notsup Unsupported platform for talib@1.0.3: wanted {"os":"darwin,linux","arch":"any"} (current: {"os":"win32","arch":"x64"})